frame/ethereum/src/lib.rs

```diff
@@ -171,7 +213,7 @@ impl<T: Trait> Module<T> {
 					frame_system::Module::<T>::block_number()
 				)
 			),
-			gas_limit: U256::zero(), // TODO: set this using Ethereum's gas limit change algorithm.
+			gas_limit: U256::from(0), // TODO: set this using Ethereum's gas limit change algorithm.
```

Comment:
Any reason for this change ?

Reply:
Yeah I honestly think `U256::zero()` is better.

Reply:
Internally we had moved it around, I'll fix this asap!